packages/wonder-blocks-modal/components/two-column-modal/two-column-modal.test.js

```diff
@@ -74,6 +74,35 @@ describe("TwoColumnModal", () => {
         expect(modal);
     });
 
+    it("should render TwoColumnModal with custom style", () => {
+        // Arrange
+        const wrapper = mount(
+            <MediaLayoutContext.Provider
+                value={{
+                    overrideSize: undefined,
+                    ssrSize: "large",
+                    mediaSpec: MEDIA_DEFAULT_SPEC,
+                }}
+            >
+                <TwoColumnModal
+                    content="Content"
+                    sidebar="Sidebar"
+                    footer="Footer"
+                    style={{
+                        maxWidth: 300,
+                        maxHeight: 200,
+                    }}
+                />
+            </MediaLayoutContext.Provider>,
+        );
+
+        // Act
+        const modal = wrapper.find(LargeTwoColumnModal);
+
+        // Assert
+        expect(modal);
```

Comment:
This should also be `.toMatchSnapshot()` so that we can verify the width/height.

Reply:
added